.travis.yml

```diff
@@ -10,6 +10,9 @@ branches:
 before_install:
 - npm config set spin false
 - npm install -g bower coveralls pr-bumper
+- bower --version
+- npm install phantomjs-prebuilt
```

Comment:
I'm not a big fan of signing up to support phantomjs personally. For the past few years, I haven't seen any upside to running  tests in phantomjs. With the ability to launch real browsers even in headless environments with `xvfb`, and phantom normally trailing behind real browsers w.r.t. features it supports, it's been more headache than it's worth. 

I'm open to hearing a convincing argument for why we should support phantomjs, but until I do, I'd vote against using it in CI or dev.